src/main/java/io/cdap/delta/bigquery/BigQueryEventConsumer.java

```diff
@@ -623,8 +623,11 @@ public synchronized void applyDML(Sequenced<DMLEvent> sequencedEvent) throws Exc
 
     latestOffset = event.getOffset();
     latestSequenceNum = sequenceNumber;
+
     if (LOG.isTraceEnabled()) {
-      LOG.trace("DML event:{} offset: {} seq num : {}", GSON.toJson(event), latestOffset.get(), latestSequenceNum);
+      LOG.trace("DML event={}, sequenceNumber={}", GSON.toJson(event), latestSequenceNum);
+    } else if (LOG.isDebugEnabled()) {
+      LOG.debug("DML Event={}, sequenceNumber={}", event, latestSequenceNum);
```

Comment:
Nit: let's log `sequenceNumber` instead of `latestSequenceNum`, although both are equivalent at this point, but semantically it makes it a little confusing, as we just want to log event's sequence number.